.travis.yml

```diff
@@ -146,7 +144,7 @@ jobs:
         tags: true
         all_branches: true
         python: *mainstream_python
-      user: __token__
+      username: __token__
```

Comment:
https://docs.travis-ci.com/user/deployment-v2/providers/pypi/ says that the default is v1 of dpl that has `user:` and not `username:`